docs/source/kiva/overview.rst

```diff
@@ -300,10 +316,11 @@ Dashes::
 
 .. image:: images/dashes.png
 
-Before filling a path, the colour of the fill is via the ``set_fill_color()``
-method, and gradient fills can be done via the ``set_linear_gradient()`` and
-``set_radial_gradient()`` methods.  Finally, there are two different fill modes
-available:
+Before filling a path, the colour of the fill is via the
+:py:meth:`~.AbstractGraphicsContext.set_fill_color` method, and gradient fills
```

Comment:
Do you think our documentation's spelling style of "colo[u]r" should match that of the API it's describing?